pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "zsk4gm"
+version = "0.0.1"
+authors = [
+  { name="Hallie Parten", email="zsk4gm@virginia.edu" },
+]
+description = "A small example package"
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+
+[project.urls]
+Homepage = "https://github.com/"
```

Comment:
These should be your github repo path and the issues path, not a biggie, just sayin'